2022.12/alpine3.17/Dockerfile

```diff
@@ -22,17 +22,18 @@ RUN buildDeps=' \
     && export GNUPGHOME="${tmpdir}/gnupg" \
     && mkdir $GNUPGHOME \
     && apk add --no-cache --virtual .build-deps $buildDeps \
+    && apk add --no-cache readline \
```

Comment:
This was added as starting the REPL would spew out errors re: missing libreadline.so.8